src/DropdownMenu.js

```diff
@@ -97,16 +97,16 @@ class DropdownMenu extends React.Component {
       ...props,
     } = this.props;
 
-    delete props.bsRole;
+    const [bsProps, elementProps] = splitBsPropsAndOmit(props, ['bsRole']);
```

Comment:
I think so, if only to help prevent it being accidentally passed through. and for folks using these utils for their custom components as well